iree/compiler/Conversion/BUILD

```diff
@@ -12,18 +12,64 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+load("//build_tools/bazel:tblgen.bzl", "gentbl_cc_library")
+
 package(
     default_visibility = ["//visibility:public"],
     features = ["layering_check"],
     licenses = ["notice"],  # Apache 2.0
 )
 
+gentbl_cc_library(
+    name = "Passes_inc_gen",
```

Comment:
The naming style here is a bit strange to me. Is it intentional? Why not just `PassesIncGen` to be consistent with others?

Reply:
Just used to the style in other projects. Updated.